python/cuml/experimental/cluster/robust_single_linkage.pyx

```diff
@@ -0,0 +1,256 @@
+#
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# distutils: language = c++
+
+from libc.stdint cimport uintptr_t
+from libcpp cimport bool
+
+from cython.operator cimport dereference as deref
+
+import numpy as np
+import cupy as cp
+
+from cuml.common.array import CumlArray
+from cuml.common.base import Base
+from cuml.common.doc_utils import generate_docstring
+from cuml.raft.common.handle cimport handle_t
+from cuml.common import input_to_cuml_array
+from cuml.common.array_descriptor import CumlArrayDescriptor
+from cuml.common.mixins import ClusterMixin
+from cuml.common.mixins import CMajorInputTagMixin
+
+from cuml.metrics.distance_type cimport DistanceType
+
+from .hdbscan_plot import SingleLinkageTree
```

Comment:
This import seems off? Probably remaining from prior things?

Reply:
I had started building the robust single linkage and abandoned it in the meantime while we get the initial impl of hdbscan out the door. I'm going to remove it from this PR and add to another.